levelObjects/Actions.md

```diff
@@ -164,7 +166,8 @@ Additional Visual Effects:
     0004  int32      Tile Y
     0008  int16      Target floor height
     000A  int16      Target ceiling height
-    000C  [4]byte    Unknown
+	000C  int16      Speed
```

Comment:
I've set this field to various values - both from what can be found in the archive.dat and other test values -- never did the movement change its speed. Either this is dependent on something else to work, or it's not working at all.